composer.json

```diff
@@ -14,15 +18,18 @@
     ],
     "require": {
         "ext-curl": "*",
-        "ext-json": "*"
+        "ext-json": "*",
+        "symfony/console": "^4"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
-        "friendsofphp/php-cs-fixer": "^2.0"
+        "phpunit/phpunit": "^7",
+        "friendsofphp/php-cs-fixer": "^2.0",
+        "phpstan/phpstan": "^0.12.67"
     },
     "autoload": {
         "psr-0": {
-            "stojg": "src/"
+            "Silverstripe": "src/",
+            "Silverstripe/PingdomCLI": "src/PindomCLI"
```

Comment:
Not sure whether you need both of these, I would remove line 32 and confirm you've spelt the directory correctly. Composer should then automatically pick up the namespace

Reply:
PSR-4 states your directory name must match the namespace. In your code you're using `Silverstripe\Pingdom`, so the directory should be `src/Pingdom`

Reply:
Actually now that I think about it, I think you need:

```json
"autoload": {
        "psr-0": {
                "Silverstripe": "src/"
        }
}
```
and structure you code in:
```
src/Silverstripe/Pingdom/Exception/
src/Silverstripe/Pingdom/Api/Client.php
src/Silverstripe/Pingdom/Command/Checks/Get.php
```

Reply:
doesn't the following
```
"autoload": {
        "psr-0": {
                "Silverstripe": "src/"
        }
}
```
 alias "src/" as "silverstripe"?